dockerfiles/Dockerfile.tensorrt

```diff
@@ -5,7 +5,7 @@
 # Dockerfile to run ONNXRuntime with TensorRT integration
 
 # nVidia TensorRT Base Image
-FROM nvcr.io/nvidia/tensorrt:20.07.1-py3
+FROM nvcr.io/nvidia/tensorrt:20.09-py3
 MAINTAINER Vinitra Swamy "viswamy@microsoft.com"
```

Comment:
>Vinitra Swamy "viswamy@microsoft.com [](start = 11, length = 36)

Should we update the MAINTAINER since Vinitra is away?

Reply:
I think @jywu-msft was saying that there is not enough use for these dockerfiles, so they are not kept up to date. I just used it temporarily for a quicker start, but will be moving to a different dockerfile eventually.